package.json

```diff
@@ -59,14 +58,12 @@
     "big.js": "^5.0.3",
     "cids": "~0.5.3",
     "datastore-core": "~0.4.0",
-    "datastore-fs": "~0.4.2",
-    "datastore-level": "~0.7.0",
+    "datastore-fs": "github:ipfs/js-datastore-fs#fix/node10",
+    "datastore-level": "github:ipfs/js-datastore-level#fix/node10",
     "debug": "^3.1.0",
     "interface-datastore": "~0.4.2",
     "ipfs-block": "~0.7.1",
-    "level-js": "timkuijsten/level.js#idbunwrapper",
-    "leveldown": "^1.7.2",
-    "lock-me": "^1.0.3",
+    "lock-me": "github:jacobheun/lock-me#fix/windows",
```

Comment:
Can we get this PR up so that @dignifiedquire can review and merge?

Reply:
We've already gotten that merged and released with lock-me 1.0.4. 

`datastore-level` is the only thing left to be merged, and then I will update all the deps for this. I will get the [feedback](https://github.com/ipfs/js-datastore-level/pull/7#discussion_r191051427) for that resolved so we can get it merged in.